package.json

```diff
@@ -5,16 +5,16 @@
   "dependencies": {
     "@atlaskit/tree": "^7.0.2",
     "@braintree/sanitize-url": "^6.0.0",
-    "@chakra-ui/react": "^1.8.3",
+    "@chakra-ui/react": "^2.0.0",
     "@chakra-ui/theme": "^2.0.3",
     "@chakra-ui/theme-tools": "^2.0.1",
     "@chakra-ui/utils": "^2.0.1",
     "@craco/craco": "^6.4.3",
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@hookform/resolvers": "^2.8.2",
-    "@opengovsg/design-system-react": "0.0.8",
-    "@sentry/react": "^5.30.0",
+    "@opengovsg/design-system-react": "0.0.12",
+    "@sentry/react": "^7.12.1",
     "@sentry/tracing": "^6.15.0",
```

Comment:
Does this need to be upgraded too? [From their migration docs](https://github.com/getsentry/sentry-javascript/blob/master/MIGRATION.md#upgrading-from-6x-to-7x):

> If you installed additional Sentry packages, such as`@sentry/tracing` alongside your Sentry SDK (e.g. `@sentry/react` or `@sentry/node`), make sure to upgrade all of them to version 7.

Reply:
good catch! i only saw the [issue](https://github.com/getsentry/sentry-javascript/issues/4904) w/ the corresponding [PR](https://github.com/getsentry/sentry-javascript/pull/4966) and just ran `npm i` so i missed this. thanks for the catch!